frontend/src/hooks/useScenes.ts

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+
+type Scene = {
+  id: number;
+  name: string;
+};
+
+export function useScenes(movieId: string | number | undefined) {
```

Comment:
Great that you used custom hooks for fetching data!

Some improvements to this hooks could be:
- Add error state management
- Use SWR or React Query for caching, error handling and revalidation
- Add types for returned data